templates/repo/sub_menu.tmpl

```diff
@@ -2,12 +2,12 @@
 	<div class="ui two horizontal center link list">
 		{{if and (.Repository.UnitEnabled $.UnitTypeCode) (not .IsBareRepo)}}
 			<div class="item{{if .PageIsCommits}} active{{end}}">
-				<a href="{{.RepoLink}}/commits{{if .IsViewBranch}}/branch{{else if .IsViewTag}}/tag{{else if .IsViewCommit}}/commit{{end}}/{{EscapePound .BranchName}}"><i class="octicon octicon-history"></i> <b>{{.CommitsCount}}</b> {{.i18n.Tr "repo.commits"}}</a>
+				<a href="{{.RepoLink}}/commits{{if .IsViewBranch}}/branch{{else if .IsViewTag}}/tag{{else if .IsViewCommit}}/commit{{end}}/{{EscapePound .BranchName}}"><i class="octicon octicon-history"></i> <b>{{.CommitsCount}}</b> {{if eq .CommitsCount 1}}{{.i18n.Tr "repo.commit"}}{{else}}{{.i18n.Tr "repo.commits"}}{{end}}</a>
```

Comment:
Please use `TrN` for translations that depend on count.
`{{.i18n.Tr (TrN .i18n.Lang .CommitsCount "repo.commit" "repo.commit") }}`